cmd/cgroups-playground/main.go

```diff
@@ -54,5 +57,37 @@ func xmain() error {
 	for i, s := range subsystems {
 		logrus.Infof("Subsystem %d: %q", i, s.Name())
 	}
+
+	cpuCgroup, err := v2.NewCpu(unifiedMountpoint)
+	if err != nil {
+		return err
+	}
+	cpuGroupPath := v2.GroupPath(filepath.Join(unifiedMountpoint, "cpu_v2"))
+	var period, shares uint64 = 1000, 5000
+	resources := specs.LinuxResources{
+		CPU: &specs.LinuxCPU{Period: &period, Shares: &shares},
+	}
+	err = cpuCgroup.Create(cpuGroupPath, &resources)
+	if err != nil {
+		return err
+	}
+	defer removePath(string(cpuGroupPath))
+	stats := stats2.Metrics{
+		CPU: &stats2.CPUStat{
+			Usage: &stats2.CPUUsage{},
+		},
+	}
+	err = cpuCgroup.Stat(cpuGroupPath, &stats)
+	if err != nil {
+		return err
+	}
+	logrus.Infof("CPU usage stats: usage in kernel mode - %d", stats.CPU.Usage.Kernel)
```

Comment:
This isn't interesting without adding process to the current group.
Why not just use the current group?

Reply:
What do you mean, by using current group?

Reply:
`g, err := v2.PidGroupPath(pid)` in the top of the function

Reply:
make sense, ok.